tools/discovery/web/traceroute/HTTP-Traceroute.py

```diff
@@ -0,0 +1,328 @@
+#!/usr/bin/env python
```

Comment:
`traceroute` directory was removed in an earlier commit. Why did you add it again?